main.go

```diff
@@ -51,9 +54,10 @@ type milestoneAPI struct {
 	UpdatedAt   *time.Time `json:"updated_at"`
 	StartDate   string     `json:"start_date"`
 	DueDate     string     `json:"due_date"`
+	Number      int        `json:"number"`
 }
 
-// Struct to get ID from main API
+// Struct for GitLab API
 type gitLabAPI struct {
```

Comment:
You use gitlabURL etc. everywhere, but here you use gitLabAPI. Small inconsistency.

Reply:
Thanks, I renamed it to gitlabAPI for better consistency.